$NetBSD$

Use GnuPG (gpg1) by default.
Prepare pkgsrc PREFIX for both, gpg and gpgv.

--- control/pgpverify.in.orig	2019-02-07 15:02:22.000000000 +0000
+++ control/pgpverify.in
@@ -140,19 +140,19 @@
 # the value of $INN::Config::gpg will override this.  On a recent Debian
 # variant, use /usr/bin/gpg1 (from the gnupg1 package) if you want to
 # support old signatures with MD5 digest algorithms.
-# $gpg = '/usr/local/bin/gpg';
+$gpg = 'PKGSRC_PREFIX/gpg';
 
 # Path to the GnuPG gpgv binary, if you have GnuPG.  If you do, this will
 # be used in preference to PGP.  For most current control messages, you
 # need a version of GnuPG that can handle RSA signatures.  If you have INN
 # and the script is able to successfully include your INN::Config module,
 # the value of $INN::Config::gpgv will override this.
-# $gpgv = '/usr/local/bin/gpgv';
+#$gpgv = 'PKGSRC_PREFIX/gpgv';
 
 # Path to pgp binary; for PGP 5.0, set the path to the pgpv binary.  If
 # you have INN and the script is able to successfully include your
 # INN::Config module, the value of $INN::Config::pgp will override this.
-$pgp = '/usr/local/bin/pgp';
+# $pgp = '/usr/local/bin/pgp';
 
 # If you keep your keyring somewhere that is not the default used by pgp,
 # uncomment the next line and set appropriately.  If you have INN and the
